### PR TITLE
fix: on overflowing .app-content div when on create

### PR DESCRIPTION
**Pull request by alex-krasn**
This is a test: first commit to repo + small bugfix on #192

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -118,6 +118,7 @@
       flex-grow: 1;
       display: flex;
       flex-direction: column;
+      overflow-y: scroll;
```
Comment (kunzms) [inline src/App.scss:121]: Let's see a demo for this change and we need to test the change in Edge browser.
